document/CoreFormat.mdk

```diff
@@ -356,65 +356,60 @@ source to allow the run to be repeated.
 A `run` object should contain a property named `files` whose value is a JSON object,
 each of whose properties represents a file that was scanned in the course of the run.
 
-Each property name in the `files` object shall be the URI of a scanned file.
-No two of these property names shall be equivalent as defined in &sect;[#uriProperties].
-
-Each property value in the `files` object shall be an array of `file` objects (&sect;[#file])
-which contain information about the file identified by the URI in the property name.
-
 The object specified by the `files` property should contain properties representing at least those files in which results were detected,
 but it may contain properties representing all scanned files (whether or not results were detected in those files),
-or for any subset of those files.
+or any subset of those files.
 
 ~ IsoNumberedNote { @IsoNumberedNote=1}
 `file` objects contain information that is useful for viewers.
 Viewers will be able to provide the most information to users if the `files`
 property is present and contains information for every file in which results were detected.
 ~
 
-If the scanned file is not nested within another file, then the array shall consist of a single
-element, and the property name shall be the URI of the scanned file.
-If the absolute location of the scanned file is available, the URI should be an absolute URI;
-otherwise, the URI shall be a relative URI.
-
 ~ IsoNumberedExample { @IsoNumberedExample=1 }
 
 ```
-run: {
```

Comment:
"`run`" does not appear in the format; each `run` object is an array element. In any case, I found it clearer to limit the example to the `files` property itself (which was missing its surrounding quotes).